back/app/controllers/web_api/v1/projects_controller.rb

```diff
@@ -20,17 +20,10 @@ def index
     # scope.
 
     @projects = Project.where(id: publications.select(:publication_id))
+      .ordered
```

Comment:
The search from `projects_controller` was removed since it turns out we don't really need to search through individual projects, but `AdminPublications` (to include projects and folders)